### PR TITLE
[sweep:integration] Replace RPCClient with Client

### PR DESCRIPTION
**Pull request by chaen**
Sweep #5761 `Replace RPCClient with Client` to `integration`.

Adding original author @chaen as watcher.

BEGINRELEASENOTES
*All
CHANGE: use Client instead of RPCClient

ENDRELEASENOTES
Closes #1095391213

### CODE REVIEW

**Comment by fstagni** [comment]
Looks like there's some circular imports.

**Comment by chaen** [comment]
yes. Having such a basic component as the `TornadoBaseClient` pull so much in terms of dependency is worrying in my opinion. I'll certainly not attempt to really fix it, but I'll try to hide it at least

File: src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py [hunk 1]
```diff
@@ -51,8 +51,6 @@
 from DIRAC.Core.Utilities import Network
 from DIRAC.Core.Utilities.JEncode import decode, encode
 
-from DIRAC.Resources.IdProvider.IdProviderFactory import IdProviderFactory
-from DIRAC.FrameworkSystem.private.authorization.utils.Tokens import getLocalTokenDict, writeTokenDictToTokenFile
```
Comment (TaykYoku) [inline src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py:55]: I don't think you need to hide it, I'd better try to rewrite it. The first thing that comes to mind is to describe these objects at the DIRAC.Core.Security level. what do you think about it?

Comment (chaen) [reply src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py:55]: Honestly, it is very hard for me to give you an opinion. As you know, I spent quite some time reviewing what you did, but I did not go to that level of detail, so I can't give an educated answer.
I think though that until you fix it, hiding it there is a good trade off ?

Comment (TaykYoku) [reply src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py:55]: OK, I'll think about a possible solution later and do a PR or issue

**Review by TaykYoku — APPROVED** [review]

**Pull request merged**